docs/units-and-global-variables.rst

```diff
@@ -311,14 +311,17 @@ For more information, see the section on :ref:`address`.
     semantics than ``delegatecall``.
 
 
-.. index:: this, selfdestruct
+.. index:: this, selfdestruct, super
 
 Contract Related
```

Comment:
Should this be changed to `Contract-related` to be consistent with the section name in the cheatsheet?

Reply:
Sure, why not. That would be the correct way to phrase it.